code/src/Components/Album.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Artists } from './Artists.js';
+
+export const Album = (props) => {
+    console.log(props);
+    return (
+        <>
+            <div className="album-container">
+                <div className="image-container">
+                    <img className="albumcover" src={props.image} alt="Album cover" />
+                    <div className="icon-container hidden">
+                        <img className="heart-icon" src="/icons/heart.svg" alt="Like-bottom"/>
+                        <img className="play-icon" src="/icons/play.svg" alt="Play"/>
+                        <img className="dots-icon" src="/icons/dots.svg" alt="Menue"/>
+                    </div>
+                </div>
+                <div className="text-container">
+                    <a href={props.albumURL} 
+                        target="_blank" 
+                        rel="noopener noreferrer"
+                    >
+                        <h2 className="album-name">{props.albumName}</h2>
+                    </a>
+                    <div className="artist-container">
+                    {
+                        props.artists.map(artist => (
+                            <Artists artistName={artist.name} artistURL={artist.external_urls.spotify}/>
+                        ))
+                    }
+                    </div>
+                </div>
+            </div>
+        </>
```

Comment:
If your component returns only one root element, there is no need to wrap it around React Fragment.

In this case WE NEED to use React Fragment :

```jsx
return (
	<div>My element 1</div>
  <div>My element 2</div>
);
```

so, after necessary changes it looks like this :

```jsx
return (
	<>
		<div>My element 1</div>
	  <div>My element 2</div>
	</>
);
```

But in this case WE DON'T NEED to use React Fragment :

```jsx
return (
	<div>My element 1</div>
);
```